components/TextFields/src/MDCTextInputControllerBase.m

```diff
@@ -956,6 +961,17 @@ + (void)setFloatingPlaceholderNormalColorDefault:(UIColor *)floatingPlaceholderN
                                                : [self class].inlinePlaceholderColorDefault;
 }
 
+- (UIColor *)floatingPlaceholderErrorActiveColor {
+  return _floatingPlaceholderErrorActiveColor;
+}
+
+- (void)setFloatingPlaceholderErrorActiveColor:(UIColor *)floatingPlaceholderErrorActiveColor {
+  if (![_floatingPlaceholderErrorActiveColor isEqual:floatingPlaceholderErrorActiveColor]) {
```

Comment:
Color equality is really weird on iOS (hence all of our new test helpers for comparing colors) so I would suggest not bothering with a call to `-isEqual:`. I would just set it :D

Reply:
That makes sense. However I saw other places in this file use this pattern, so I think it's better to keep it for consistency sake.